t/99-full-stack.t

```diff
@@ -62,6 +63,8 @@ like $log, qr/wait_still_screen: detected same image for 1 seconds/, 'test type
 like $log, qr/wait_still_screen: detected same image for 0\.1 seconds/, 'test type string and wait for .1 seconds';
 like $log, qr/.*event.*STOP/, 'Machine properly paused';
 like $log, qr/.*event.*RESUME/, 'Machine properly resumed';
+like $log, qr/Saving storage devices \(Current VM state is running\)/, 'save_storage started';
```

Comment:
**FYI** This is failing in GitHub Actions:

    doesn't match '(?^u:Saving storage devices \(Current VM state is running\))'